cirq/google/calibration/phased_fsim.py

```diff
@@ -76,3 +85,102 @@ def override_by(self, other: 'PhasedFSimParameters') -> 'PhasedFSimParameters':
             Otherwise the current values are used.
         """
         return other.merge_with(self)
+
+
+@json_serializable_dataclass(frozen=True)
+class FloquetPhasedFSimCalibrationOptions:
+    characterize_theta: bool
+    characterize_zeta: bool
+    characterize_chi: bool
+    characterize_gamma: bool
+    characterize_phi: bool
+
+
+# TODO: Fix json serialization (the default one doesn't work with tuples as dictionary keys).
+# TODO: Add start and end calibration timestamp
```

Comment:
I would just create issues for these and leave them for later.  Myself or someone else can help out with this.

Reply:
OK, I can do that later.